cppsdk/include/playfab/PlayFabEvent.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#ifndef DISABLE_PLAYFABENTITY_API
+
+#include <playfab/PlayFabError.h>
+#include <playfab/PlayFabEventsDataModels.h>
+
+namespace PlayFab
+{
+    enum struct PlayFabEventServerProcessingType
```

Comment:
>PlayFabEventServerProcessingType [](start = 16, length = 32)

Should we just call it PlayFabEventType ?

Reply:
Sure! That was my first thought too, but the name looked way too generalized

---
In reply to: [227943924](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227943924) [](ancestors = 227943924)